app/mailers/notification_mailer.rb

```diff
@@ -1,9 +1,9 @@
 class NotificationMailer < ActionMailer::Base
   default from: "noreply@firehose-defcon.com"
 
-  def send_notification(new_post, recipient)
-    @new_post = new_post
-    @recipient = recipient
+  def send_notification(new_post_id, recipient_id)
```

Comment:
Made this change to call by IDs to follow Sidekiq best practices (to prevent a stale instance from having around in the queue)